{"name": "Load more", "status": "broken", "statusDetails": {"message": "TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'", "trace": "request = <SubRequest 'driver' for <Function test_upload_1error_file>>\n\n    @pytest.fixture(scope=\"function\", autouse=True)\n    def driver(request):\n        options = Options()\n        perfs = {\"download.defoult_directory\": f\"{os.getcwd()}/downloads\"}\n        options.add_argument(\"--headless\")\n        options.add_experimental_option(\"excludeSwitches\", ['enable-automation'])\n        options.add_experimental_option(\"useAutomationExtension\", False)\n        options.add_experimental_option(\"prefs\", perfs)\n        options.add_argument(\"--window-size=1920,1080\")\n        options.add_argument(\"--no-sandbox\")\n        options.add_argument(\"--disable-dev-shm-usage\")\n        options.add_argument(\"--ignore-certificate-errors\")\n        #service = Service(f\"{os.getcwd()}/chromedriver\")\n        #service.start()\n>       driver = webdriver.Chrome(executable_path=(f\"{os.getcwd()}/chromedriver\"), service=Service(ChromeDriverManager().install()), options=options)\nE       TypeError: WebDriver.__init__() got an unexpected keyword argument 'executable_path'\n\nconftest.py:26: TypeError"}, "start": 1718832759753, "stop": 1718832759753, "uuid": "03911fe7-feed-4032-8077-5d2a812265e8", "historyId": "b876855393093e4f206ee3077aea6915", "testCaseId": "b876855393093e4f206ee3077aea6915", "fullName": "tests.test_bev.TestBEV#test_upload_1error_file", "labels": [{"name": "feature", "value": "BEV Functionality"}, {"name": "severity", "value": "Critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_bev"}, {"name": "subSuite", "value": "TestBEV"}, {"name": "host", "value": "c22506624fdb"}, {"name": "thread", "value": "1-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_bev"}]}